README.md

```diff
@@ -112,7 +112,7 @@ In addition to the standard connection parameters the driver supports a number o
 | binaryTransfer                | Boolean |          true           | Use binary format for sending and receiving data if possible. Setting this to false disables any binary transfer                                                                                                                                                                                                                              |
 | binaryTransferEnable          | String |           ""            | Comma separated list of types to enable binary transfer. Either OID numbers or names                                                                                                                                                                                                                                                          |
 | binaryTransferDisable         | String |           ""            | Comma separated list of types to disable binary transfer. Either OID numbers or names. Overrides values in the driver default set and values set with binaryTransferEnable.                                                                                                                                                                   |
-| prepareThreshold              | Integer |            5            | Statement prepare threshold. A value of -1 stands for forceBinary                                                                                                                                                                                                                                                                             |
+| binaryTransferBufferSize      | Integer |          4096           | Size of buffers for copying inputs to outputs in case of OID creation using Blobs.                                                                                                                                                                                                                                                            |
```

Comment:
Does it relate to all the binary transfers or for blobs only?
If the parameter is related to blobs only, I would expect the name to contain something like blob or large object or something like that.

I suggest we start tracking `@since 42.7.1` so the reader can see which parameters are supported in what versions

Reply:
Its only for blobs, yes. `binaryTransferBlobBufferSize`?